ENIGMAsystem/SHELL/Widget_Systems/Cocoa/dialogs.m

```diff
@@ -150,7 +149,7 @@ int cocoa_show_error(const char *str, bool abort)
     return result;
 }
 
-const char *cocoa_get_open_filename(const char *title, const char *filter, const char *fname, const char *dir, const bool mselect)
+const char *cocoa_get_open_filename(const char *filter, const char *fname, const char *dir, const char *title, const bool mselect)
```

Comment:
Why change the argument order? It caused a lot of diff, and if you missed any references in this change, they'll now be broken.

Reply:
I didn't break anything. This is fully tested in GM. I found it more logical than the original setup. Because I want the Obj-C functions to have the same argument order as the C++.

Reply:
I find your answer to his question acceptable, but please understand the point he is making. Frivolous changes like this also polluted the git blame, just like whitespace changes do. In terms of git history it is always best to get it right the first time rather than having to come back and tweak the function signatures.